README.md

```diff
@@ -19,15 +19,28 @@ Follow these instructions to set up your local development environment. Only Mac
 1. **[Docker](https://www.docker.com/products/docker-desktop)** - containerization Kubernetes cluster on your desktop.
    - :exclamation: **After install:** Enable local Kubernetes cluster in preferences per [these instructions](https://docs.docker.com/desktop/kubernetes/#enable-kubernetes).
      It will take a few minutes to provision the cluster for the first time.
-1. **[Homebrew](https://brew.sh/)** - package manager for MacOs.
-1. **[NodeJs (LTS)](https://nodejs.org/docs/latest-v14.x/api/index.html) and NPM** -
+2. **[Homebrew](https://brew.sh/)** / **[Chocolatey](https://community.chocolatey.org/)** - package manager for macOS / Windows.
+3. **[NodeJs (LTS)](https://nodejs.org/docs/latest-v14.x/api/index.html) and NPM** -
    Follow [NVM Wiki](https://github.com/SolomonDefi/solomon-monorepo/wiki/NVM) to install
    and select the correct Node/NPM versions
+4. **[Helm](https://helm.sh/)** - The package manager for Kubernetes.
 
-1. **[Skaffold](https://skaffold.dev/)** - build/deploy tool for local Kubernetes development.
+   ```sh
+   # macOS
+   $ brew install helm
+
+   # Windows (with admin PowerShell)
+   $ chocolatey install helm
```

Comment:
I prefer to keep this out of the Readme - it implies we will officially support windows developers, which is not the case.